docs/standard/microservices-architecture/implement-resilient-applications/monitor-app-health.md

```diff
@@ -1,50 +1,50 @@
 ---
 title: Health monitoring
-description: .NET Microservices Architecture for Containerized .NET Applications | Health monitoring
+description: Explore one way of implementing health monitoring.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 12/11/2017
+ms.date: 10/16/2018
 ---
 # Health monitoring
 
 Health monitoring can allow near-real-time information about the state of your containers and microservices. Health monitoring is critical to multiple aspects of operating microservices and is especially important when orchestrators perform partial application upgrades in phases, as explained later.
 
-Microservices-based applications often use heartbeats or health checks to enable their performance monitors, schedulers, and orchestrators to keep track of the multitude of services. If services cannot send some sort of “I’m alive” signal, either on demand or on a schedule, your application might face risks when you deploy updates, or it might simply detect failures too late and not be able to stop cascading failures that can end up in major outages.
+Microservices-based applications often use heartbeats or health checks to enable their performance monitors, schedulers, and orchestrators to keep track of the multitude of services. If services cannot send some sort of “I’m alive” signal, either on demand or on a schedule, your application might face risks when you deploy updates, or it might just detect failures too late and not be able to stop cascading failures that can end up in major outages.
 
-In the typical model, services send reports about their status, and that information is aggregated to provide an overall view of the state of health of your application. If you are using an orchestrator, you can provide health information to your orchestrator’s cluster, so that the cluster can act accordingly. If you invest in high-quality health reporting that is customized for your application, you can detect and fix issues for your running application much more easily.
+In the typical model, services send reports about their status, and that information is aggregated to provide an overall view of the state of health of your application. If you're using an orchestrator, you can provide health information to your orchestrator’s cluster, so that the cluster can act accordingly. If you invest in high-quality health reporting that's customized for your application, you can detect and fix issues for your running application much more easily.
 
-## Implementing health checks in ASP.NET Core services
+## Implement health checks in ASP.NET Core services
```

Comment:
This change causes an anchor link to here to be broken from docs/standard/microservices-architecture/architect-microservice-container-applications/resilient-high-availability-microservices.md

Reply:
Fixed with PR https://github.com/dotnet/docs/pull/10015

Can you have them merged in the same "transaction"?